main.js

```diff
@@ -20,16 +20,44 @@ define(function (require, exports, module) {
         Launcher             = require("lib/Launcher").Launcher,
         NoHostServer         = require("nohost/src/NoHostServer").NoHostServer;
 
-    var _server = new NoHostServer({
-        pathResolver    : ProjectManager.makeProjectRelativeIfPossible,
-        root            : ProjectManager.getProjectRoot().fullPath
-    });
+    var _server;
 
     function _getServer() {
+        if (!_server) {
+            _server = new NoHostServer({
+                pathResolver    : ProjectManager.makeProjectRelativeIfPossible,
+                root            : ProjectManager.getProjectRoot()
+            });
+        }
         return _server;
     }
 
-    // First, we configure brackets to run the experimental live dev
+    // We wait until the LiveDevelopment module is initialized and the project loaded
+    // so we can safely swap our transport and launcher modules for
+    // the defaults and start LiveDev.
+    function _configureLiveDev() {
+        // Turn preview iFrame On
+        Browser.init();
+
+        function _configureModules() {
+            // Set up our transport and plug it into live-dev
+            PostMessageTransport.setIframe(Browser.getBrowserIframe());
```

Comment:
Actually, let's use `*Iframe` everywhere, to match what Brackets code does (i.e., capital `I` and lowercase `f`)